stablehlo/reference/Element.cpp

```diff
@@ -538,6 +564,38 @@ Element min(const Element &e1, const Element &e2) {
       });
 }
 
+Element power(const Element &e1, const Element &e2) {
+  Type type = e1.getType();
+
+  if (isSupportedIntegerType(type)) {
+    APInt base = e1.getIntegerValue();
+    APInt exponent = e2.getIntegerValue();
+    bool isSigned = base.isSignedIntN(base.getBitWidth());
+    if (isSigned && exponent.isNegative()) {
+      if (base.isZero())
+        llvm::report_fatal_error("divide by zero unsupported");
```

Comment:
Let's drop these two lines of code to simplify the interpreter? Per the spec, corner cases like this one are implementation-defined, so we might as well define our implementation to be as simple as possible.

Reply:
This would mean 0 raised to negative power would evaluate to 0.

Reply:
I think it would be best to produce some value rather than crashing. If we have two big tensors and just one of the elementwise pairs is zero base and negative exponent, it is useful to continue evaluation rather than crash the entire thing.